docs/cli-return-codes.md

```diff
@@ -0,0 +1,17 @@
+
+# Command Line Interface (CLI) Return Codes
+
+The NeoFS CLI returns specific exit codes to indicate the outcome of command execution.
+
+## Exit Codes
+
+| Exit Code | Meaning                                   |
+|-----------|-------------------------------------------|
+| 0         | Command executed successfully.           |
+| 1         | Internal error or an unspecified failure.|
+| 2         | Object access denied or unauthorized.    |
+| 3         | Await timeout expired for a certain condition. |
```

Comment:
can we justify the table? (my ide does that automatically)

Reply:
e.g. https://github.com/nspcc-dev/neofs-node/blob/27623c50edbb86a09e9b4b368be3e3d399a92969/docs/storage-node-configuration.md?plain=1#L15-L26

Reply:
done